vars/githubscm.groovy

```diff
@@ -74,6 +74,38 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def tagRepository(String repository, String author, String branch, String tagUserName, String tagUserEmail, String tagName) {
+    checkout(resolveRepository(repository, author, branch, false))
+    def currentCommit = getCommit()
+    println """
+-------------------------------------------------------------
+[INFO] Tagging ${author}/${repository}:${branch}
+-------------------------------------------------------------
+Commit: ${currentCommit}
+Tagger: ${tagUserName} (${tagUserEmail})
+Tag: ${tagName}
+-------------------------------------------------------------
+"""
+    sh("""
+        git config user.name '${tagUserName}'
+        git config user.email '${tagUserEmail}'
+        git tag -a ${tagName} -m 'Tagging ${tagName}.'
+    """)
+
+    try {
+        withCredentials([usernamePassword(credentialsId: 'kie-ci, usernameVariable: 'GIT_USERNAME', passwordVariable: 'GIT_PASSWORD')]){
```

Comment:
there's a typo here `credentialsId: 'kie-ci` should be `credentialsId: 'kie-ci'`